### DISCUSSION TITLE
Contradictory CWA Store availability

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3541] [Development] [answered]
I assume this will sort itself out, but currently https://www.coronawarn.app/en/faq/results/#ramp_down_stores says:

"The CWA will be available in the app stores of Apple and Google until mid June 2023."

whereas https://play.google.com/store/apps/details?id=de.rki.coronawarnapp says:

"As of June 1, 2023, the Corona-Warn-App will be placed in a hibernation mode and will no longer be available for download in the stores."

and https://apps.apple.com/de/app/corona-warn-app/id1512595757 says:

"Ab dem 1. Juni 2023 wird die Corona-Warn-App in einen Ruhemodus versetzt und wird nicht mehr in den Stores zum Download zur Verfügung stehen."

**Comment by Ein-Tim** [comment]
The app will be removed this week.

**Reply by MikeMcC399** [reply]
@Ein-Tim 

Thanks for your confirmation. When the app is removed from the Google and Apple stores, the problem is solved. 🙂

**Comment by Ein-Tim** [comment, marked as answer]
The app has been removed now and we will begin the process of removing all references to the stores.

**Reply by MikeMcC399** [reply]
@Ein-Tim 
- The link https://apps.apple.com/de/app/corona-warn-app/id1512595757 to Apple now displays "Verbindung zu Apple Music wird hergestellt" and then nothing happens
- The link https://play.google.com/store/apps/details?id=de.rki.coronawarnapp displays the app still, however if you click on "Install on more devices" it says "This item isn't available"

so yes, it makes sense to remove the references to the store links now.

**Reply by Ein-Tim** [reply]
Thanks for checking Mike, and for your info, it might take some time until all references are removed.

**Reply by MikeMcC399** [reply]
@Ein-Tim It sounds like you are the only one remaining on duty. This is probably also a good time for me to follow the exit of others. So I will just monitor only if I am mentioned now.
All the best to you!

**Reply by Ein-Tim** [reply]
@MikeMcC399 

Thank you again, very very much, for all your work and only the best to you too! ❤️
